android/app/src/main/java/io/peggyjo/peggymeter/HistoryTextFragment.java

```diff
@@ -43,7 +43,8 @@ private void renderLog(View view, List<LogEntry> logs) {
                 getResources().getString(R.string.smile2),
                 getResources().getString(R.string.smile3),
                 getResources().getString(R.string.smile4)};
-        table.setAdapter(new TableRowAdapter(getContext(), R.layout.listview_item_row, reverse(logs), smileys));
+        table.setAdapter(new TableRowAdapter(getContext(), R.layout.listview_item_row,
+                reverse(logs), smileys));
```

Comment:
Can you align reverse(logs) with the first arg above: "new TableRowAdapter...".

Reply:
done